build/references/coreth-arc20s.md

```diff
@@ -0,0 +1,103 @@
+
+#### Native Assets - Tokens
+
+Non-AVAX Assets, however, have no counterpart within the EVM. Therefore, Coreth makes minor modifications to the EVM and state management in order to support asset balances and the ability to transfer these assets on the C-Chain.
+
+Non-AVAX Assets (ANTs) are supported on the C-Chain by keeping an account balance mapping [assetID -> balance]. These assets can be exported back to the X-Chain or moved around on the C-Chain using `assetCall` and `assetBalance`.
+
+##### assetCall
+
+`assetCall` is a precompiled contract at address `0xassetCallAddr`. `assetCall` allows users to atomically transfer a native asset to a given address and (optionally) also make a contract call to that same address. This is parallel to how a normal transaction can send value to some address, and atomically call that address with some `data`.
+
+
+```text
+assetCall(address addr, uint256 assetID, uint256 assetAmount, bytes memory callData) -> {ret: bytes memory}
+```
+
+These arguments can be packed by `abi.encodePacked(...)` in Solidity since there is only one argument with variadic length (callData). The first three arguments are constant length, so the precompiled contract simply parses the call `data` input as:
+
+
+```text
++-------------+---------------+--------------------------------+
+| address     : address       |                       20 bytes |
++-------------+---------------+--------------------------------+
+| assetID     : uint256       |                       32 bytes |
++-------------+---------------+--------------------------------+
+| assetAmount : uint256       |                       32 bytes |
++-------------+---------------+--------------------------------+
+| callData    : bytes memory  |            len(callData) bytes |
++-------------+---------------+--------------------------------+
+                              |       84 + len(callData) bytes |
+                              +--------------------------------+
+```
+
+##### assetBalance
+
+`assetBalance` is a precompiled contract at address `0xassetBalance`. `assetBalance` is the ANT equivalent of using `balance` to get the AVAX balance.
+
+```text
+assetBalance(address addr, uint256 assetID) -> {balance: uint256}
+```
+
+These arguments can be packed by `abi.encodePacked(...)` in Solidity since all of the arguments have constant length.
+
+```text
++-------------+---------------+-----------------+
+| address     : address       |        20 bytes |
++-------------+---------------+-----------------+
+| assetID     : uint256       |        32 bytes |
++-------------+---------------+-----------------+
+                              |        52 bytes |
+                              +-----------------+
+```
+
+## ARC20s on the C-Chain
+
+An ARC20 is the Avalanche equivalent of an ERC20. An ERC20 is just a smart contract that maintains balances for accounts and provides an incredibly useful, standardized interface for smart contract developers.
```

Comment:
Recommend removing "just" to make tone neutral.

"An ERC20 is **just** a smart contract that maintains balances" to "n ERC20 is a smart contract that maintains balances"

Reply:
Updated

Reply:
It's difficult to describe the difference without using some modifier. I changed it to describing ERC-20s as second class citizens in the world of Ethereum (because they are stored on the smart contract instead of on the account that owns them), but added the qualifier that the rich interface they create enables the powerful world of DeFi.